packages/phrase/package.json

```diff
@@ -15,12 +18,11 @@
     "csv-stringify": "^6.2.3",
     "debug": "^4.3.1",
     "form-data": "^4.0.0",
-    "node-fetch": "^2.6.1",
     "picocolors": "^1.0.0"
   },
   "devDependencies": {
     "@types/debug": "^4.1.5",
-    "@types/node-fetch": "^2.5.7"
+    "@types/node": "^20.11.22"
```

Comment:
If the `engines` config specifies 18, should the types be set to 18 to ensure you dont rely on v20 features accidentally?

Reply:
Not sure on this one. Ideally we'd do a matrix test on CI to test all LTS versions above 18 or something, but we barely use anything modern anyway, other than `fetch`. Happy to go either way though, I'm not against downgrading.

Reply:
Downgrade to node 18